packages/app/src/components/header/TheHeader.vue

```diff
@@ -305,10 +305,6 @@ const hasContent = computed(() => {
   .hero-banner-container {
     @apply absolute left-0 top-full flex h-64 w-full items-end justify-end overflow-hidden bg-primary-900;
 
-    &.mainnet {
```

Comment:
This hack was needed to increase the height of the header background (which has a fixed size) when a warning message is shown. Otherwise the header text is pushed down and is not visible:
<img width="419" alt="image" src="https://github.com/matter-labs/block-explorer/assets/6553665/7b2e8fa5-d60a-4b0d-952c-e2022232b39a">
Having dynamic error messages we will also need to change the height of the background image dynamically or rework the whole thing.

Reply:
We generally don't support mobile, but this should be fixed properly. Out of scope of this PR I'd say.